lib/brexit_checker/convert_csv_to_yaml/actions_processor.rb

```diff
@@ -34,6 +36,12 @@ def parse_logic_fields(record)
         end
       end
 
+      def parse_grouping(record)
+        group_criteria = record["result_groups"]
```

Comment:
This is a bit confusing. maybe don't use a function here and just use
```stripped_record["result_groups"] &&= stripped_record.split(",").map(&:strip)```

Reply:
would that work if it's nil?